hack/release/releasenotes/make-release-notes.go

```diff
@@ -0,0 +1,145 @@
+// Copyright Project Contour Authors
```

Comment:
This can probably just be folded into the existing `prepare-release.go`, no need to have it be a separate tool. Looking into that change now.

Reply:
OK, second commit moves it into prepare-release.go so it can all be run in one shot.